packages/core/test/invoke.test.ts

```diff
@@ -1784,44 +1782,48 @@ describe('invoke', () => {
             type: 'parallel',
             states: {
               first: {
-                invoke: {
-                  src: () => () => {
-                    throw new Error('test');
-                  },
-                  onError: {
-                    target: 'failed',
-                    cond: () => {
-                      errorHandlersCalled++;
-                      return false;
+                initial: 'waiting',
```

Comment:
this test was weirdly written so I've refactored it in the process, this should land on the main branch regardless of this PR - I will create a branch with just this simple adjustment later on